### ISSUE TITLE
Feature Suggestion: "Retry" button for prompts in History

### ISSUE THREAD

**Issue by donuts-are-good** [divamgupta/diffusionbee-stable-diffusion-ui#135]
Right now in the history, you cannot right click select and copy the words, and you don't have a button to "re-render" the photo using the same settings. If you want to run a historical prompt again you have to manually type out the prompt characteristics into a notepad, then paste that into the main window text-to-image input (which does not have right click copy paste either).

Bonus points if you can click retry on a history item and change the seed or other parameters before the next render.

**Comment by barryanders** [comment]
Retry is great. I was just coming to make this request and here you already made it. Making the history text selectable so it can be copied would be nice too. I find myself often wanting to copy the text in history so I can fine tune it for another go and I can't.

**Comment by tveskov** [comment]
Agreed, one-click regeneration would be fantastic. Also, in the currect version the history view jumps back to the most current image when one goes to eg. "Text to Image" so copy/pasting words/parameters from history becomes a drag.
